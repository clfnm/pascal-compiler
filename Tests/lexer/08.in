*A
A*
*****
.A..
..4
.4
5.
5..
A.A
a+b
?&@!
'v
7
"b7"''
и