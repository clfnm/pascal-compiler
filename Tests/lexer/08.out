1	1	Id	A	A
1	2	Eof	EOF	